sample-extension-plugin/build.gradle

```diff
@@ -49,6 +49,36 @@ dependencies {
     }
 }
 
+// DynamoDB Local setup for integration tests
+task startDynamoDBLocal(type: Exec) {
+    group = 'verification'
+    description = 'Start DynamoDB Local container'
+    commandLine 'docker', 'run', '-d', '--name', 'dynamodb-local-test', '-p', '8000:8000', 'amazon/dynamodb-local'
+    ignoreExitValue = true
+}
+
+task stopDynamoDBLocal(type: Exec) {
+    group = 'verification'
+    description = 'Stop and remove DynamoDB Local container'
+    commandLine 'docker', 'rm', '-f', 'dynamodb-local-test'
+    ignoreExitValue = true
+}
+
+task integTestWithRemoteStore(type: RestIntegTestTask) {
```

Comment:
Can we also add a Github workflow similar to https://github.com/opensearch-project/job-scheduler/blob/main/.github/workflows/multi-node-test.yml for this?

Reply:
I removed the changes in `sample-extension-plugin/build.gradle` for now as I need to set up the DyanmoDB Local setup for integration tests separately. I will need to solve the issue of access denied when accessing aws credentials.